script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -127,37 +127,37 @@ I turned around, facing the voice behind me.
 There was a nurse in front of the door with a disturbed look on her face.
 <Page36>
 "How did you break the bed, Shiki-kun?"
-He kept on asking me "how" I broke the bed, not so much as why I did it.
+He kept on asking me 'how' I broke the bed, not so much as why I did it.
```

Comment:
Haven't playtested this so I'm not sure but it seems like its the nurse saying this not the doctor.